nodejs/make-data-key.js

```diff
@@ -1,22 +1,17 @@
-const { readMasterKey, CsfleHelper } = require("./helpers")
+const kms = require("./kms");
+require("dotenv").config();
 
 async function main() {
-  const localMasterKey = readMasterKey()
+  const kmsClient = kms.localCsfleHelper();
 
-  const csfleHelper = new CsfleHelper({
-    kmsProviders: {
-      local: {
-        key: localMasterKey
-      }
-    }
-  })
+  const unencryptedClient = await kmsClient.getRegularClient();
 
-  const client = await csfleHelper.getRegularClient()
-
-  const dataKey = await csfleHelper.findOrCreateDataKey(client)
-  console.log("Base64 data key. Copy and paste this into clients.js\t", dataKey)
-
-  client.close()
+  const dataKey = await kmsClient.findOrCreateDataKey(unencryptedClient);
+  console.log(
+    "Base64 data key. Copy and paste this into clients.js\t",
```

Comment:
Nitpick: I wonder if a linefeed would be better than a tab just to make the start and end as clear as possible.